build.sbt

```diff
@@ -30,5 +41,4 @@ lazy val settings = Seq(
     val required = "1.8"
     val current  = sys.props("java.specification.version")
     assert(current == required, s"Unsupported JDK: java.specification.version $current != $required")
-  }
-)
+  }) ++ formattingSettings
```

Comment:
It doesn't seem like there's a particularly good way to include the build files per se in the reformatting set. The nearest I can find is something like:
http://stackoverflow.com/questions/19201509/how-to-format-the-sbt-build-files-with-scalariform-automatically